src/Web/Web.csproj

```diff
@@ -23,11 +25,6 @@
 		<None Include="$(SpaRoot)\**" Exclude="$(SpaRoot)\node_modules\**" />
 	</ItemGroup>
 
-	<ItemGroup>
-		<!-- Extends watching group to include Angular app changes -->
```

Comment:
Why was this removed?

Reply:
Is there a way to keep this check in place when we're using the `no-proxy` launch target? That way users running `dotnet watch` without Visual Studio will allow their environment to reload on any changes.